src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -49,3 +49,182 @@
 .tribe-theme-genesis .tribe-block__event-website a {
 	font-size: 1.65rem;
 }
+
+/* = Venue block view styles
+============================================= */
+.tribe-block__venue {
+	border-top: 1px solid #d5d5d5;
```

Comment:
are you using variables or hardcoding the color values?

Reply:
We should be using a variable for this. I've made the update.

Reply:
there are other color variables in this PR, please check and update those.

Reply:
I've updated the last remaining one, thanks.